loganalytics/data-plane/README.md

```diff
@@ -0,0 +1,102 @@
+# Azure Log Analytics
+
+This project provides client tools or utilities in Java that make it easy to query data in [Azure Log Analytics](https://azure.microsoft.com/en-us/services/log-analytics/). For reference documentation on classes and models, please see the [Azure SDK for Java reference](https://docs.microsoft.com/en-us/java/api/overview/azure/?view=azure-java-stable). 
+
+Azure Log Analytics provides agents for telemtry collection and enables deep analytics via a [rich query language](https://docs.loganalytics.io/index). This SDK provides query access to data already stored in Log Analytics. To start collecting data from different sources, take a look at these [quickstarts](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-quick-collect-azurevm). 
+
+## Example
```

Comment:
We probably don't want the example to be in the readme because if the number of examples grow the readme will keep growing. 

You can follow Storage team's examples here: https://github.com/Azure/azure-storage-java-async/blob/master/samples/src/main/java/com/microsoft/azure/storage/samples/BasicSample.java